frontend/Styleguide/Icon.tsx

```diff
@@ -0,0 +1,44 @@
+export function Icon({ variant, size = 16, alt }: IconProps) {
+  const IconComponent: React.FunctionComponent<SpecificIconProps> =
+    IconComponents[variant];
+
+  if (!IconComponent) {
+    throw Error(`Unknown IconVariant '${variant}'`);
+  }
+
+  return <IconComponent size={size} alt={alt} />;
+}
+
+export interface IconProps {
+  variant: string;
+  alt?: string;
+  size?: number;
+}
+
+type SpecificIconProps = Omit<IconProps, "variant">;
+
+export enum IconVariant {
+  close = "close",
+}
+
+const IconComponents = {
+  [IconVariant.close]: CloseIcon,
+};
+
+export function CloseIcon({ size, alt }: SpecificIconProps) {
+  return (
+    <svg
```

Comment:
Very interesting snippet to create the close icon.  Tailwind has a locked close and open icon at https://heroicons.com/ in svg and jsx.

Reply:
Ah maybe we should switch to heroicons then. Let's investigate that with Jessie when we create future icons - the one I added will need to match whichever icon set we choose.